src/encoded/static/components/visualizations.js

```diff
@@ -1,5 +1,63 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { ResultsTable } from './regulome_search';
+import { isLight } from './datacolors';
+
+export const mapChromatinNames = name => (
+    name.includes('TssAFlnk') ? 'Flanking Active TSS'
+    : name.includes('TssA') ? 'Active TSS'
+    : name.includes('TxFlnk') ? "Transcr. at gene 5' and 3'"
+    : name.includes('TxWk') ? 'Weak transcription'
+    : name.includes('Tx') ? 'Strong transcription'
+    : name.includes('EnhG') ? 'Genic enhancers'
+    : name.includes('EnhBiv') ? 'Bivalent Enhancer'
+    : name.includes('Enh') ? 'Enhancers'
+    : name.includes('ZNF/Rpts') ? 'ZNF genes & repeats'
+    : name.includes('Het') ? 'Heterochromatin'
+    : name.includes('TssBiv') ? 'Bivalent/Poised TSS'
+    : name.includes('BivFlnk') ? 'Flanking Bivalent TSS/Enh'
+    : name.includes('ReprPCWk') ? 'Weak Repressed PolyComb'
+    : name.includes('ReprPC') ? 'Repressed PolyComb'
+    : name.includes('Quies') ? 'Quiescent/Low'
+    : null
+);
```

Comment:
I've changed this and think it should be good, but some of the state abbreviations that we are using may not match the ones on my reference and then I may need to update again. For instance, we have "1_TssA" instead of "TssA" and I got around this by using 'includes'